src/Header/Account/Account.jsx

```diff
@@ -166,17 +166,23 @@ function Account(props) {
 					Object.keys(accountsData.links).map(function (text, i) {
 						return (
 							<li key={i} role="presentation">
-								<a
-									href={accountsData.links[text]}
-									role="menuitem"
-									onClick={handleMenuItemClick}
-									onKeyDown={handleKeyDown}
-									data-hj-suppress={
-										accountsData.links[text].indexOf("profile") > -1 ? "" : null
-									}
-								>
-									{text}
-								</a>
+								{accountsData.links[text] != "#" ? (
```

Comment:
I'm not a fan of using a hash as the thing that says it's a link or not. I think it would be better to use the `display_name` prop directly, rather than duplicating it into the links array.

Reply:
Yeah I didn't either but it appears that's how nice accounts is implemented but have now changed it to use display_name for idam. Apparently Accounts and Idam return JSON differently and I think that's where the code is a bit inconsistent.